### PR TITLE
hiding ui based on permission

### PR DESCRIPTION
**Pull request by beaesguerra**

### CODE REVIEW

File: src/services/Token/index.ts [hunk 1]
```diff
@@ -57,7 +57,7 @@ const TokenService: TokenService = {
     // -- END OVERRIDE -- //
     // Update listeners
     R.map(
-      (fn: Listener) => fn(value),
+      (fn: Listener) => setTimeout(R.partial(fn, [value]), 0),
```
Comment (Right2Drive) [inline src/services/Token/index.ts]: Silly person that wrote this before, making your work so much harder :P

File: src/views/pages/ManagePeople/index.vue [hunk 2]
```diff
@@ -1,7 +1,11 @@
 <template>
   <div class='ManagePeople'>
     <div class="add-button">
-      <md-button @click="onAddClick" class="md-fab md-primary md-fab-bottom-right md-fixed add-user-button">
+      <md-button 
+        @click="onAddClick" 
+        class="md-fab md-primary md-fab-bottom-right md-fixed add-user-button"
+        v-if="this.$store.state.user.type === 'admins'"
+        >
```
Comment (Right2Drive) [inline src/views/pages/ManagePeople/index.vue]: I think this is formatted wrong?

Comment (beaesguerra) [reply src/views/pages/ManagePeople/index.vue]: fixed

File: src/services/Token/index.ts (hunk 1 above)
Comment (beaesguerra) [reply src/services/Token/index.ts]: haha thanks for helping me with this bug

**Pull request merged**